utilities/backupable/backupable_db.cc

```diff
@@ -2248,8 +2248,14 @@ Status BackupEngineImpl::BackupMeta::StoreToFile(bool sync) {
   std::unique_ptr<char[]> buf(new char[max_backup_meta_file_size_]);
   size_t len = 0, buf_size = max_backup_meta_file_size_;
   len += snprintf(buf.get(), buf_size, "%" PRId64 "\n", timestamp_);
+  if (len >= buf_size) {
+    return Status::Corruption("Buffer too small to fit backup metadata");
```

Comment:
iiuc, we use `Corruption` to denote data corruption on db. This case does not seem to fall into that category.

Reply:
Line 2267 had ```return Status::Corruption("Buffer too small to fit backup metadata");``` .  So I added same message here as well as other places. Should I change the error Status to something else?

Reply:
I see. In this case, I think we can keep it for now until we revisit this later.